docs/installation.rst

```diff
@@ -14,31 +14,48 @@ Installing ``openfe``
 When you install ``openfe`` through any of the methods described below, you
 will install both the core library and the command line interface (CLI). 
 
-Installation with ``conda`` (recommended)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Installation with ``micromamba`` (recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We recommend installing ``openfe`` with ``conda``, because it provides easy
+We recommend installing ``openfe`` with `micromamba <https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html#micromamba>`_, because it provides easy
 installation of other tools, including molecular dynamics tools such as
-OpenMM and ambertools, which are needed by ``openfe``. 
-Conda can be installed either with the `full Anaconda distribution
-<https://www.anaconda.com/products/individual>`_, or with
-the `smaller-footprint miniconda
-<https://docs.conda.io/en/latest/miniconda.html>`_.
+OpenMM and ambertools, which are needed by ``openfe``.
 
-You can get ``openfe`` from the ``conda-forge`` channel. To install the most
-recent release in its own environment, use the commands ::
+To install ``micromamba`` execute the `installation script <https://mamba.readthedocs.io/en/latest/installation.html#install-script>`_.
+We recommend ``micromamba`` because it is faster than ``mamba`` and ``conda`` and does not require a ``base`` environment. 
 
-  $ conda create -n openfe -c conda-forge openfe
-  $ conda activate openfe
+On Linux the default shell is ``bash`` ::
+  
+  $ curl micro.mamba.pm/install.sh | bash
+
+And on macOS the default shell is ``zsh`` ::
+
+  $ curl micro.mamba.pm/install.sh | zsh
+
+Now we will set the channel where packages are downloaded from to ``conda-forge`` ::
+
+  $ micromamba config append channels conda-forge
+
+This command will create an environment called ``openfe_env`` with the ``openfe`` package and all required  dependencies ::
+
+  $ micromamba create -n openfe_env openfe
+
+Now we need to activate our new environment ::
+
+  $ micromamba activate openfe_env
+
+To make sure everything is working, run the tests ::
 
-That creates a separate environment for ``openfe`` that won't interfere with
-other things you have installed. You will need to activate the ``openfe``
-environment before using it in a new shell.
+  $ pytest --pyargs openfe
```

Comment:
```suggestion
  $ pytest --pyargs openfe openfecli
```
? (can't remember if you also need a second pyargs before openfecli

Reply:
Just tested it and you don't need a second pyargs

Reply:
done!

Reply:
This'll just turn into the OpenFE test command anyway